health route redesign (#565)
meilisearch/documentation: references/health.md

# Health

> The health check endpoint enables the health of a MeiliSearch instance to be periodically tested.

Reviewer: ```suggestion
The health check endpoint enables you to periodically test the health of your MeiliSearch instance.
```
